### ISSUE TITLE
Write a "What to Expect" when you moderate section

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutSPACEs#249]
## Summary

Prepare moderators for the worst case scenario, and make sure they have support.

**Comment by SuziQpid** [comment]
> Following co-working with @skfantoni :
> 
> * As an AutSPACEs moderator, you will be in a position to shield others from potentially upsetting, triggering, explicit or abusive content.
> * This means you will have to be prepared to read that content
> * You must be psychologically robust
> * Once you an experience to moderate it you have 24 hours and then it goes back
> * The amount of time you put in is up to you
> * You’ll get a snippet view and if you don’t want to moderate/look at it further you don’t have to respond
> * Why is moderation important?
> * Emphasise why it matters - incentivise moderating
> * List of skills that you gain as a moderator
> * Following code of conduct
> * You can get support from AutSPACEs researchers?
> * Meet other moderators? (aspect of community?) - every 3 months moderators can meet - offering courses? - building a community is important
> * Transfer to support page
> * Worth asking Autistica?
> * Insight group is a good outreach path

Hi just to say language needs to be a little clearer in 4 th bullet point  eg: ‘ Once you HAVE ( word missing) an experience to moderate, you have 24 hrs to review it before it’s returned to the repository’ 


I don’t think it’s the job of researchers to support moderators unless they want to straddle two roles.
l think it would be a good idea to have a moderators community Meetup bi or 3  monthly with input from lnsight members who feel able to participate / were part of initial focus groups/ have had problems using the repository. It would always be a good idea to have at least two reps at that Meetup who can comment on moderating the third party reported experiences.

**Comment by GeorgiaHCA** [comment]
Thanks @SuziQpid ! I've clarified the fourth bullet point. I think your point about researchers supporting moderators is a really excellent one and good food for thought. On the one hand I do think researchers have a responsibility to ensure the wellbeing of moderators they work with, but I also don't want that to be at the expense of autistic moderators having genuine power in building a safe, welcoming community. Worth digging into so we can get it right. Maybe a good topic for a meet-up session?

**Comment by gedankenstuecke** [comment]
This seems to now be a duplicate of #250, so i'm closing down this one :)
